crates/matrix-sdk-base/src/store/traits.rs

```diff
@@ -846,6 +877,9 @@ pub enum StateStoreDataKey<'a> {
 
     /// Recently visited room identifiers
     RecentlyVisitedRooms(&'a UserId),
+
+    /// Composer draft for the room
```

Comment:
Can you elaborate what it is? Imagine you don't know what the editor is, what the composer is, what the draft is, and you've to explain that here :-). Also, you can create links to `ComposerDraft` and `DraftType` maybe, like so:

```rust
/// To learn more, see [`ComposerDraft`] and [`DraftType`].
///
/// [`ComposerDraft`]: path::to::ComposerDraft
/// [`DrafType`]: path::to::DraftType
```